config/config.go

```diff
@@ -37,6 +39,8 @@ const (
 	NetworkKey = "NETWORK"
 	// BaseAssetKey is the default asset hash to be used as base asset for all markets. Default is LBTC
 	BaseAssetKey = "BASE_ASSET"
+	//NativeAssetHash is used to set lbtc hash, used for fee outputs, in regtest network
+	NativeAssetHash = "NATIVE_ASSET_HASH"
```

Comment:
Just looking at it now,let be consistent

```suggestion
	NativeAssetHash = "NATIVE_ASSET"
```

Reply:
Done